tests/testthat/test-tag.R

```diff
@@ -1,6 +1,6 @@
 test_that("tag works", {
 
-  tag_check <- shinyGovstyle::tag_Input("tag1", "COMPLETE")
+  tag_check <- shinyGovstyle::tag_Input("tag1", "Complete")
   expect_identical(
     tag_check$children[[1]],
     "COMPLETE"
```

Comment:
check here fails - probably just needs changing to lower case here but run devtools::test() to make sure tests pass